Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -923,7 +923,7 @@ void CGameListCtrl::OnMouseMotion(wxMouseEvent& event)
       GetItemRect(item, Rect);
       int mx = Rect.GetWidth();
       int my = Rect.GetY();
-#ifndef __WXMSW__
+#if !defined(__WXMSW__) && !defined(__WXMAC__)
```

Comment:
I think WXMAC should be __WXOSX__

According to the api docs: "old define, same as __WXOSX__"